tutorial-queries/codeql-pack.yml

```diff
@@ -1,4 +1,5 @@
 name: getting-started/tutorial-queries
 version: 0.0.1
+dbscheme: csv.dbscheme
 dependencies:
   codeql/tutorial: ^0.0.1
```

Comment:
We probably want to widen this dependency range since we may be putting out newer versions of the tutorial pack later.

Reply:
I've tested that the query still works with `>=0.0.3`